packages/client/hooks/useScrollThreadList.ts

```diff
@@ -43,7 +46,8 @@ const useScrollThreadList = (
         // the delay is required for new task cards, not sure why height is determined async
       }, 50)
     }
-  }, [isInit, threadables, preferredNames])
+    setOldClientHeight(clientHeight)
+  }, [isInit, threadables, preferredNames, el?.clientHeight])
```

Comment:
oh now we're getting into fun stuff!
accessing the `clientHeight` seems really harmless because it's just a property. In reality, it triggers a DOM repaint.
in other words, it's like
```ts
class Element {
  get clientHeight() {
    return this.blockTheThreadAndTalktoTheDOM()
  }
}
```

by doing this on every re-render we can no longer leverage the speed of the VDOM.

perhaps you need `useLayoutEffect`?